src/material-experimental/theming/_m3-tokens.scss

```diff
@@ -464,12 +482,30 @@
     ),
     _namespace-tokens(
       (mdc, fab),
-      mdc-tokens.md-comp-fab-primary-values($systems, $exclude-hardcoded),
+      (
+        mdc-tokens.md-comp-fab-primary-values($systems, $exclude-hardcoded),
+        (
+          // Color variants
+          primary: (), // Default, no overrides needed.
+          secondary: mdc-tokens.md-comp-fab-secondary-values($systems, $exclude-hardcoded),
+          tertiary: mdc-tokens.md-comp-fab-tertiary-values($systems, $exclude-hardcoded),
+          error: () // TODO(mmalerba): Should we add our own error variant?
```

Comment:
Is an "error FAB" a thing?

Reply:
I can't really think of a good use for it, which is why I ask. For things like form-field, checkbox, etc. its possible for them to be in an invalid state, so it makes sense to use the error variant for that. I can't really think of a case like that for FABs which is probably why MDC didn't make tokens for an error FAB. On the other hand, we have historically allowed people to do an error FAB if they want

Reply:
Let's not support the error variant for M3 since it seems to be unsupported by Material Design anymore. Since we still provide it for M2 then we don't need to consider this breaking